src/main/java/net/javadiscord/javabot/systems/help/model/HelpTransaction.java

```diff
@@ -0,0 +1,19 @@
+package net.javadiscord.javabot.systems.help.model;
+
+import lombok.Data;
+
+import javax.annotation.Nullable;
+import java.time.LocalDateTime;
+
+/**
+ * Data class that represents a single Help Transaction.
+ */
+@Data
+public class HelpTransaction {
+	private long id;
+	private long recipient;
+	private LocalDateTime createdAt;
+	private double value;
+	@Nullable
+	private String message;
```

Comment:
As stated previously, I don't think that message is necessary. Instead we could use an enum for the type of transaction and a user id of the user causing the transaction.The message could then be derived when necessary (from the transaction type, )